README.md

```diff
@@ -28,19 +28,17 @@ As a pet sitter, you can provide the service of walks using your own schedule, p
 
 ## Installation
 
-**Pre-requisites**: 
-- installed Xcode with the version specified under the [Current development Environment](#current-dev-environment).
-
-1) clone the current repository by running the following command in shell:
-```bash
-$ git clone git@github.com:ios-course/ironfoudation-team-project.git
-```
-If you need more information, please check the [GitHub's documentation](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
-
-2) open the file `Woof.xcodeproj` in Xcode.
-3) select your device as the destination.
-4) press the run button.
-If you don't know how to run the app on a real device, please check the [documentation](https://developer.apple.com/documentation/xcode/running-your-app-in-simulator-or-on-a-device).
+**Pre-requisites**: _installed Xcode with the version specified under the [current development environment](#current-dev-environment)._
+
+1. Go to the [Release](https://github.com/ios-course/ironfoudation-team-project/releases) section in the current repository.
+2. Select the latest one or any specific release you want to install.
```

Comment:
You can use the following numeration to make it more convenient and to avoid issues:
```suggestion
1. Go to the [Release](https://github.com/ios-course/ironfoudation-team-project/releases) section in the current repository.
1. Select the latest one or any specific release you want to install.
1. ...
```

Reply:
Please update the list as we have discussed.

Reply:
The list are updated.